servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -321,6 +321,10 @@ def cromwell_query_params(query, page, page_size):
     query_params.append({'additionalQueryResultFields': 'parentWorkflowId'})
     query_params.append({'additionalQueryResultFields': 'labels'})
     query_params.append({'includeSubworkflows': 'false'})
+    if query.extensions:
```

Comment:
nit: single if:
```
if query.extensions and query.extensions.hide_archived:
```
(can switch back to two if/when more query extensions are needed or cromwell)

Reply:
k, done